src/main/scala/mesosphere/marathon/state/ResourceRole.scala

```diff
@@ -1,5 +1,34 @@
 package mesosphere.marathon.state
 
+import com.wix.accord._
+import com.wix.accord.dsl._
+import mesosphere.marathon.api.v2.Validation._
+
 object ResourceRole {
   val Unreserved = "*"
+
+  val validResourceRole: Validator[String] = {
+    val message = "A role name must not be %s."
+    isTrue[String](message.format("\"\"")) { role => role != ""; } and
```

Comment:
Not sure why the formatting is a bit funky here, but this is what `sbt test` seems to like...